pkg/metrics/metrics.go

```diff
@@ -0,0 +1,110 @@
+package metrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/labstack/echo"
+	"github.com/shreyasj2006/goyagi/pkg/config"
+)
+
+type statsdClient interface {
+	Histogram(name string, value float64, tags []string, rate float64) error
+	Count(name string, value int64, tags []string, rate float64) error
+}
+
+// Metrics functions for metrics clients.
+type Metrics struct {
```

Comment:
FYI since this is a place where Goyagi isn't completely up to date -- we moved a lot of this code out into https://github.com/lob/metrics-go since we were repeating it across several repos!

Reply:
Ah nice! Thanks for the heads up!